src/test/scripts/functions/builtin/residencymatch.dml

```diff
@@ -0,0 +1,10 @@
+# this dml file will run the  ResidencyMatchMain function in builtin functions  in scripts/builtin
+R = read($1)
+H = read($2)
+C = read($3)
+alg = $4
+if( alg == "residencymatch" )
```

Comment:
please remove this condition as you have only one algorithm to match.

Reply:
done